app/modules/booking/view_bookings/viewBookings.controller.js

```diff
@@ -6,13 +6,13 @@ const viewBookings = async(req, res) => {
         if (validation.isEmpty())
             return res.send(validation);
 
-        const result = await viewBookingsQuery(req.query.search_key, req.query.sort_key || 'firstName', req.query.sort_order || 'asc');
-
+        let result = await viewBookingsQuery(req.query.search_key, req.query.sort_key || 'firstName', req.query.sort_order || 'asc', req.query.page || global.DEFAULT_PAGINATION_LIMIT, req.query.pagesize || 0)
```

Comment:
Function parameters should be minimized for better coding standard, you can assign a param object and include all the parameters inside the object and pass that object to the query file to ensure more readability.